package.json

```diff
@@ -6,7 +6,9 @@
   "type": "module",
   "files": [
     "app/javascript/**/*.js",
-    "plugin.js"
+    "app/views/**/*.{arb,erb,html,rb}",
+    "plugin.js",
+    "vendor/javascript/flowbite.js"
```

Comment:
is this missing the newly vendored rails-ujs?

Reply:
I've only included files that are required for compiling tailwind.

Reply:
This was updated in #8221